src/clients/save_network.rb

```diff
@@ -259,16 +259,41 @@ def configure_dns
       DNS.create_hostname_link
     end
 
+    # Creates target's /etc/hosts configuration
+    #
+    # It uses hosts' configuration as defined in AY profile (if any) or
+    # proceedes according the proposal
+    def configure_hosts
+      configured = false
+      configured = NetworkAutoYast.instance.configure_hosts if Mode.autoinst
+      NetworkAutoconfiguration.instance.configure_hosts if !configured
+    end
+
+    def configure_routing
+      return if !Mode.autoinst
+
+      NetworkAutoYast.instance.configure_routing
+    end
+
+    def configure_nics
+      return if !Mode.autoinst
+
+      NetworkAutoYast.instance.configure_nics
+    end
+
     # It does an automatic configuration of installed system
```

Comment:
upgrade scenario ignores network setup ... but i'm not sure about autoupgrade - will check